packages/pipeline-editor/src/pipelineProperties.ts

```diff
@@ -36,6 +42,19 @@ const pipelineProperties = {
         parameter_ref: 'runtime',
         label: { default: 'Pipeline Runtime' }
       },
+      {
+        control: 'custom',
+        custom_control_id: 'StringControl',
+        parameter_ref: 'schedule_interval',
+        label: {
+          default:
+            'Pipeline Schedule Interval(CRON expression, once if left empty)'
+        },
+        data: {
+          placeholder: '0 0 * * *',
```

Comment:
If possible (albeit more work), maybe we can make this more intuitive to use by include the airflow presets as an option via dropdown in addition to supporting the cron job notation e.g. left column here:
![image](https://user-images.githubusercontent.com/13340013/148130030-e8fb7d10-cc31-47f8-9f35-44544cae49d4.png)